README.md

```diff
@@ -14,9 +14,17 @@
 
 - [Installation](#installation)
 - [Usage](#usage)
-  - [Signed URLs](#signed-urls)
-  - [Srcset Generation](#srcset-generation)
+- [Signed URLs](#signed-urls)
+- [Srcset Generation](#srcset-generation)
+  - [Fixed Width Images](#fixed-width-images)
+    - [Variable Quality](#variable-quality)
+  - [Variable Size Images](#variable-size-images)
```

Comment:
From an ordering standpoint, I think we should order the subsections as:
1. Fluid Width Images
2. Fixed Width Images

to put more focus on the more common use case (fluid).

Reply:
I think I'm in complete agreement on using "Width" in both cases. I need to think about "fluid"